### DISCUSSION TITLE
Warn Sail users to prefer `npm run dev` over `sail npm run dev` for Laravel Bootcamp

### DISCUSSION THREAD

**Discussion by Johannes-Werbrouck** [laravel/framework#44323] [Ideas]
When following the Laravel Bootcamp, I chose Sail to get everything running using docker. This caused issues when I tried to run `sail npm run dev`, Vite was not able to return anything so just a blank page was displayed. When I stopped the `sail npm run dev` command, the regular pages were displayed correctly again. 
I was able to work around this by running `npm run dev` on my local machine as opposed to running it in the container. As I see it there are two possible improvement options here: 

1. Troubleshoot further and get `sail npm run dev` to work. I will need help for this. 
2. Add a warning in the docs for Sail users, so they run `npm run dev` locally. For this, I can create a PR.

**Comment by JPantanal** [comment]
I had a similar issue.  I spent some time trying to figure out what I was doing wrong. Thanks.
